package.json

```diff
@@ -59,8 +59,7 @@
     "@types/express": "^4.11.1",
     "@types/node": "^9.4.6",
     "debug": "^3.1.0",
-    "google-auth-library": "^1.6.1",
-    "googleapis": "^39.2.0"
+    "google-auth-library": "^5.10.0"
```

Comment:
This version of the library only supports Node 8 and up. This library supports Node 6 and up.

Reply:
Node 6 has been past end of life support since April 2019. Node 8 is also past end of life support since December 2019. 

In addition, npm audit is reporting two high-level vulnerabilities in this older version of google-auth-library due to its dependency on an older version of axios and node-forge:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution in node-forge                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ node-forge                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >= 0.10.0                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ actions-on-google                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ actions-on-google > google-auth-library > gtoken >           │
│               │ google-p12-pem > node-forge                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/1561                      │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Server-Side Request Forgery                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ axios                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.21.1                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ actions-on-google                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ actions-on-google > google-auth-library > axios              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/1594                      │
└───────────────┴──────────────────────────────────────────────────────────────┘
```